SU-SWS/stanford_components
# EARTH-138: Added templates to patterns for use in paragraphs.

# READY FOR REVIEW

# Summary
- A new centered template for the CTA List paragraph
- Updates to the grid settings that match Matson

# Needed By (Date)
- End of sprint

# Urgency
- Moderate

# Steps to Test

1. Check out this branch
2. Check out EARTH-138 on stanford_paragraph_types
3. Import all the feature code `drush features-import-all`
4. Create a CTA list paragraph
5. Review centered template

# Associated Issues and/or People
- EARTH-138
- Matson has a PR
- stanford_paragraph_types has a PR

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
